Keas.Mvc/ClientApp/components/Equipment/EquipmentBigFixInfo.tsx

```diff
@@ -136,6 +136,14 @@ export default class EquipmentBigFixInfo extends React.Component<
           isFetched: true,
           isFound: false
         });
+      } else if (err.message === 'Forbidden') {
```

Comment:
@laholstege We're actually returning the proper 403 status code here -- is reading the error message the best we can do?  Is the fetch library hiding the status code on the client side somehow?

Reply:
@srkirkland our fetch in `api.ts` is swallowing it and only throwing `res.statusText`. i'm happy to change it, that's just how it is now

Reply:
@laholstege oh i see that now, i'll need to have words with the guy who wrote that.